backend/corpora/common/entities/entity.py

```diff
@@ -62,3 +62,30 @@ def __getattr__(self, name):
         :return:
         """
         return self.db_object.__getattribute__(name)
+
+    @classmethod
+    def _create_sub_objects(
+        cls, rows: typing.List[dict], db_table: Base, keys: typing.List[str] = None, **kwargs
+    ) -> typing.List[Base]:
+        """
+        Create a list of Table Rows to be added to an Entity Object during an object creation. If id is provided, then
+        the row is retrieved and updated.
+
+        :param rows: A list of dictionaries containing columns.
+        :param db_table: The Table to add or modify rows
+        :param keys: additional primary keys
+        :param kwargs: Additional columns attributes to add.
```

Comment:
Not sure what this is used for. As a general comment on this function, an example usage could be helpful in clarifying if documentation is insufficient. The way I see it is, clean code can be (in this order of importance):

- explicitly named, highly modular and thus self-explanatory
- well-documented
- example usage/tutorial provided

I think the need for each as you go down the list increases with code complexity. This, for example, is a nice, compact, but complex piece of code. My point here is, it's great, but needs to adhere to some standard of readability for the sake of maintainability and developer efficiency.